install/get-tor-osx.py

```diff
@@ -60,17 +62,15 @@ def main():
         print("Downloading {}".format(dmg_url))
         r = requests.get(dmg_url)
         open(dmg_path, "wb").write(r.content)
-        dmg_sha256 = hashlib.sha256(r.content).hexdigest()
     else:
         dmg_data = open(dmg_path, "rb").read()
-        dmg_sha256 = hashlib.sha256(dmg_data).hexdigest()
-
-    # Compare the hash
-    if dmg_sha256 != expected_dmg_sha256:
-        print("ERROR! The sha256 doesn't match:")
-        print("expected: {}".format(expected_dmg_sha256))
-        print("  actual: {}".format(dmg_sha256))
-        sys.exit(-1)
+
+
+    print("Downloading {}".format(asc_url))
```

Comment:
fstrings tho :(

Reply:
this was copy-pasta code that isn't being used because bundled tor is broken - I want to re-write the whole thing